index.src.html

```diff
@@ -199,9 +204,53 @@ <h3 id="examples">Examples</h3>
 <section>
   <h2 id="concept">Concepts</h2>
 
+  <h3 id="concept-client">Clients</h3>
+
+  A <dfn export>client</dfn> represents a particular origin's relationship to
+  a set of <a>endpoints</a>.
+
+  Each <a>client</a> has an <dfn for="client" export attribute>origin</dfn>,
+  which is an <a>origin</a>.
+
+  Each <a>client</a> has an
+  <dfn for="client" export attribute>endpoint-groups</dfn> list, which is a list
+  of <a>endpoint groups</a>, each of which MUST have a distinct
+  {{endpoint group/name}}.
+
+  <h3 id="concept-endpoint-groups">Endpoint groups</h3>
+
+  An <dfn export>endpoint group</dfn> is a set of <a>endpoints</a> that will be
+  used together for backup and failover purposes.
+
+  Each <a>endpoint group</a> has a
+  <dfn for="endpoint group" export attribute>name</dfn>, which is an ASCII
+  string.
+
+  Each <a>endpoint group</a> has an
+  <dfn for="endpoint group" export attribute>endpoints</dfn> list, which is a
+  list of <a>endpoints</a>.
+
+  Each <a>endpoint group</a> has a
+  <dfn for="endpoint group" export attribute>subdomains</dfn> flag, which is
+  either "`include`" or "`exclude`".
+
+  Each <a>endpoint group</a> has a
```

Comment:
Previously you could have different TTLs for each endpoint, this changes this to a group-wide TTL. Are we OK with that? Would you ever want to have different endpoints with different TTLs?

Reply:
Yeah, I think it makes more sense for the TTL to be per-group.  I've view the TTL as describing how long the entire configuration is valid, not how long we expect any individual endpoint to be up.  The delivery mechanism will gracefully handle if a dodgy endpoint goes down, and new header values always overwrite old ones, even if a TTL hasn't expired, so there's no worry about stale data persisting for too long.

Reply:
Makes sense. LGTM to merge from this end.